src/bundle/Resources/public/js/scripts/fieldType/ezimageasset.js

```diff
@@ -0,0 +1,240 @@
+                this.toggleLoading(false);
+                showErrorNotification(error);
+            });
+        }
+
+        /**
+         * Updates Image Asset preview data
+         *
+         * @method updateData
+         * @param {Number} destinationContentId
+         * @param {Object} image
+         */
+        updateData(destinationContentId, image) {
+            const preview = this.fieldContainer.querySelector('.ez-field-edit__preview');
+            const previewImg = preview.querySelector('.ez-field-edit-preview__media');
+            const previewActionPreview = preview.querySelector('.ez-field-edit-preview__action--preview');
+            const nameContainer = preview.querySelector('.ez-field-edit-preview__file-name');
+            const sizeContainer = preview.querySelector('.ez-field-edit-preview__file-size');
+
+            previewImg.setAttribute('src', image.uri);
+            previewActionPreview.setAttribute('href', image.uri);
+
+            nameContainer.innerHTML = image.fileName;
+            nameContainer.title = image.fileName;
+            sizeContainer.innerHTML = this.formatFileSize(image.fileSize);
+            sizeContainer.title = this.formatFileSize(image.fileSize);
+
+            this.inputDestinationContentId.value = destinationContentId;
+        }
+
+        /**
+         * Loads preview for selected Image Asset
+         *
+         * @param response
+         */
+        loadPreview(response) {
+            const imageField = response.CurrentVersion.Version.Fields.field.find((field) => {
+                return field.fieldDefinitionIdentifier === imageAssetMapping['content_field_identifier']
+            });
+
+            this.updateData(response.ContentInfo.Content._id, imageField.fieldValue);
+            this.showPreview();
+        }
+
+        /**
+         * Open UDW to select an existing Image Asset
+         *
+         * @method openUDW
+         * @param {Event} event
+         */
+        openUDW(event) {
+            const udwContainer = doc.getElementById('react-udw');
+            const config = JSON.parse(event.currentTarget.dataset.udwConfig);
+            const title = Translator.trans(/*@Desc("Select Image Asset")*/ 'ezimageasset.title', {}, 'universal_discovery_widget');
+
+            const closeUDW = () => ReactDOM.unmountComponentAtNode(udwContainer);
+            const onCancel = closeUDW;
+            const onConfirm = (items) => {
+                closeUDW();
+                this.loadPreview(items[0]);
+            };
+            const canSelectContent = ({item, itemsCount}, callback) => {
+                const itemContentType = item.ContentInfo.Content.ContentTypeInfo.identifier;
+                const isAllowedContentType = itemContentType === imageAssetMapping['content_type_identifier'];
+
+                callback(isAllowedContentType);
+            };
+
+            ReactDOM.render(React.createElement(eZ.modules.UniversalDiscovery, Object.assign(config, {
```

Comment:
I think this should be the other way, the `config` should be able to override you settings